package.json

```diff
@@ -22,10 +22,16 @@
     "coverage": "nyc report && if [ $CODACY_PROJECT_TOKEN'' != '' ] ; then nyc report --reporter=text-lcov | codacy-coverage; fi",
     "ci": "npm run build && npm run test && npm run test:headless && npm run coverage",
     "prerelease": "npm run ci",
-    "release": "standard-version -s",
+    "release": "standard-version -s -a",
```

Comment:
I personally don't like to use `-a` because I often have scratch files in my project folder that would then mistakenly be added to the repo. Doesn't the `precommit` script for `standard-version` below add the correct files anyway? Seems like maybe this isn't necessary?

Reply:
I will need to test,  but i don't think it will commit those scratch files, since according to the docs
`You will need to stage the artifacts you want to commit,`. 

i'll test to confirm though

Reply:
Just tested, and the `-a` flag will only commit things that are already staged.  So if there are working files that havent yet been staged or are not yet under source control, those won't be included in the release commit